nuntium/templates/nuntium/profiles/stats.html

```diff
@@ -0,0 +1,39 @@
+{% extends "base_edit.html" %}
+{% load i18n %}
+{% load pagination_tags %}
+{% load nuntium_tags %}
+{% load staticfiles %}
+
+{% block extrajs %}
+{% endblock extrajs %}
+
+{% block extracss %}
+<link href="https://gitcdn.github.io/bootstrap-toggle/2.2.0/css/bootstrap-toggle.min.css" rel="stylesheet">
+{% endblock extracss %}
+
+{% block extrascripts %}
```

Comment:
You're right but I think I should open a new issue for it as it is a broader thing than just this issue. What do you say?